scm/etc/scripts/MOSAiC_AMPS_forcing_file_generator.py

```diff
@@ -0,0 +1,590 @@
+
+writefile_soil_depth_dim = writefile_fid.createDimension('soil_depth', None)
+writefile_soil_depth_var = writefile_fid.createVariable('soil_depth', 'f4', ('soil_depth',))
+writefile_soil_depth_var[:] = soil_depth[:]
+writefile_soil_depth_var.units = 'm'
+writefile_soil_depth_var.description = 'soil depth'
+
+writefile_ice_thickness_dim = writefile_fid.createDimension('ice_thickness', None)
+writefile_ice_thickness_var = writefile_fid.createVariable('ice_thickness', 'f4', ('ice_thickness',))
+writefile_ice_thickness_var[:] = ice_thickness
+writefile_ice_thickness_var.units = 'm'
+writefile_ice_thickness_var.description = 'depth of ice layers'
+
+#create variables and write them out
+
+#scalar group
+
+writefile_lat_var = writefile_scalar_grp.createVariable('lat', 'f4')
+writefile_lat_var[:] = lat
+writefile_lat_var.units = 'degrees N'
+writefile_lat_var.description = 'latitude of column'
+
+writefile_lon_var = writefile_scalar_grp.createVariable('lon', 'f4')
+writefile_lon_var[:] = lon
+writefile_lon_var.units = 'degrees E'
+writefile_lon_var.description = 'longitude of column'
+
+writefile_hice_var = writefile_scalar_grp.createVariable('hice', 'f4')
+writefile_hice_var[:] = hice
+writefile_hice_var.units = 'm'
+writefile_hice_var.description = 'sea ice thickness'
+
+writefile_slmsk_var = writefile_scalar_grp.createVariable('slmsk', 'f4')
+writefile_slmsk_var[:] = slmsk
+writefile_slmsk_var.units = ''
+writefile_slmsk_var.description = 'land-sea-ice mask'
+
+writefile_tsfco_var = writefile_scalar_grp.createVariable('tsfco', 'f4')
+writefile_tsfco_var[:] = tsfco
+writefile_tsfco_var.units = 'm'
+writefile_tsfco_var.description = 'sea ice surface skin temperature'
+
+writefile_weasd_var = writefile_scalar_grp.createVariable('weasd', 'f4')
+writefile_weasd_var[:] = weasd
+writefile_weasd_var.units = 'mm'
+writefile_weasd_var.description = 'water equivalent accumulated snow depth'
+
+writefile_fice_var = writefile_scalar_grp.createVariable('fice', 'f4')
+writefile_fice_var[:] = fice
+writefile_fice_var.units = '1'
+writefile_fice_var.description = 'ice fraction'
+
+writefile_tisfc_var = writefile_scalar_grp.createVariable('tisfc', 'f4')
+writefile_tisfc_var[:] = tisfc
+writefile_tisfc_var.units = 'K'
+writefile_tisfc_var.description = 'ice surface temperature'
+
+writefile_snwdph_var = writefile_scalar_grp.createVariable('snwdph', 'f4')
+writefile_snwdph_var[:] = snwdph
+writefile_snwdph_var.units = 'mm'
+writefile_snwdph_var.description = 'water equivalent snow depth'
+
+writefile_tg3_var = writefile_scalar_grp.createVariable('tg3', 'f4')
+writefile_tg3_var[:] = tg3
+writefile_tg3_var.units = 'K'
+writefile_tg3_var.description = 'deep soil temperature'
+
+writefile_zorl_var = writefile_scalar_grp.createVariable('zorl', 'f4')
+writefile_zorl_var[:] = zorl
+writefile_zorl_var.units = 'cm'
+writefile_zorl_var.description = 'composite surface roughness length'
+
+writefile_zorll_var = writefile_scalar_grp.createVariable('zorll', 'f4')
```

Comment:
@hertneky Writing out zorll, zorlw, zorli as identical to zorl -- needed by lsm_ics.